### ISSUE TITLE
Read to end doesn't function properly in NVDA

### ISSUE THREAD

**Issue by Gene703122** [nvaccess/nvda#14637]
### Steps to reproduce:
Read a book using Kindle and read to end in NVDA.
#### NVDA installed/portable/running from source:
The current nonbeta version of NVDA, installed.  The beta, running as portable.  
#### NVDA version:
2022.4.0.27401
#### Windows version:
Windows 10 21H2
#### Name and version of other software in use when reproducing the issue:
Kindle 1.39.2
There is a new version available but I haven't upgraded in case there are unknown accessibility problems.  
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Earlier versions than the current production version don't have this problem.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I haven't run it.  The problem has been reported in an NVDA users list by others and I thought it was irrelevant to run it.  I have no symptoms of a registry problem and the prior NVDA version doesn't have the problem.

**Comment by josephsl** [comment]
If this is what I think it is, then this is a duplicate of #14390. Thanks.
